src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+// Variables
+const populationData = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+// Calculation
+const populationList = [...populationData].map(data =>
+  Number(data.innerText.replaceAll(',', '')));
+const total = populationList.reduce((amount, population) =>
+  amount + population, 0);
```

Comment:
You can combine these two methods into one reduce method.
```suggestion
const total = [...populationData].reduce((amount, population) => {
  const numberPopulation = Number(population.innerText.replaceAll(',', '')));
  
  return amount + numberPopulation;
  }, 0);
```

Reply:
Thanks :)